perf_test

```diff
@@ -152,6 +194,16 @@ if __name__ == "__main__":
     if args.bandwidth + args.iops == []:
         args.bandwidth = ['1M']
 
+    try:
+        fc = setup_funccount(args.funccount, args.read, args.write)
+    except funccount.FunccountInvalidFunctionError as e:
+        print(f'Failed to setup funccount: {e}', file=sys.stderr)
+        fc = None
+    except FileNotFoundError as e:
+        print('Could not find funccount. Please install from https://github.com/brendangregg/perf-tools/pull/115',
```

Comment:
Oh, yikes, we need the specific custom version from that pull request.... That is really not something we want as it creates a giant pain for anyone trying to use this script. If this was really necessary, I'd almost just make my own implementation of funccount in python here (it's pretty simple)... But I don't think that's necessary...

Seems like your change is not actually needed. I was a bit confused by the argument passing as this code is passing only a single argument with spaces (which eventually gets passed directly to set_ftrace_filter). Then your PR to funccount seems to just make it so it can accept multiple arguments and and pass them as one to set_trace_filter.... This doesn't seem like a requirement.


```
sudo funccount-perf 'vfs_write vfs_open'
```
Works as one would expect.

If we use the bpfcc version then we have to do:

```
sudo funccount-bpfcc  'vfs_write|vfs_open'
```

Because it actually uses a regular expresion to filter the available functions in the python script.



So let's just plan on using standard funccount tools... Not link to unecessary PRs or depend on unreleased software.